saga/seata-saga-spring/src/main/java/org/apache/seata/saga/engine/tm/DefaultSagaTransactionalTemplate.java

```diff
@@ -273,8 +273,10 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
     }
 
     @Override
-    public void cleanUp() {
-        TransactionHookManager.clear();
+    public void cleanUp(GlobalTransaction tx) {
+        if (tx == null || tx.getGlobalTransactionRole() == GlobalTransactionRole.Launcher) {
```

Comment:
I don't think it should be null, since it should be strictly judged to clean up the hook, if it's null it should throw an exception prompting the message that the value is not allowed to be null
我认为不应该为null，既然要清理hook，应该是严格判断的，如果为null应该抛出异常提示该值不允许为null的信息